bin/commandTopicConsumer.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Confluent Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+# http://www.confluent.io/confluent-community-license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+#
+# Usage: commandTopicConsumer.py [-h] [-f CONFFILE] [-b BROKERS][-k KSQLSERVICEID]
+# Command topic consumer that dumps CREATE, DROP and TERMINATE queries to
+# stdout. If no arguments are provided, default values are used. Default broker
+# is 'localhost:9092'. Default ksqlServiceId is 'default_'. You may optionally
+# provide a configuration file with broker specific configuration parameters.
+# Every run of this script will consume the topic from the beginning.
+# optional arguments:
+#   -h, --help        show this help message and exit
+#   -b BROKERS        Bootstrap servers
+#   -f CONFFILE       Configuration file (configProp=value format)
+#   -k KSQLSERVICEID  KsqlDB service ID
+
+
+from confluent_kafka import KafkaError, KafkaException, version
+from confluent_kafka import Producer, Consumer
+import json
+import logging
+import argparse
+import uuid
+import sys
+import re
+
+
+class CommandRecord (object):
+    def __init__(self, stmt):
+        self.stmt = stmt
+
+    def __str__(self):
+        return "({})".format(self.stmt)
+
+    @classmethod
+    def deserialize(cls, binstr):
+        d = json.loads(binstr)
+        return CommandRecord(d['statement'])
+
+max_offset = -1001
+def consumer_latest_offsets(consumer, partitions):
```

Comment:
nit: I think it would be cleaner to just inline this in `consumer_run` and not use a global max_offset:

```
def consumer_run(self):
    max_offset = -1001

    def latest_offsets(consumer, partitions):
        nonlocal max_offset
        ...

    self.consumer.subscribe([self.topic], on_assign=latest_offsets)
```

why?
    - better encapsulation - `consumer_latest_offsets` and `max_offset` are just used by `consumer_run`
    - the global namespace is shared with everything you import - so there's a chance it could clash w/ something else.